src/ncurses/window.cr

```diff
@@ -97,6 +105,14 @@ module NCurses
       LibNCurses.wtimeout(self, value)
     end
 
+    def add_char(chr, position = nil)
```

Comment:
here you could use the `Point` record mentioned above and access `x` and `y` with its getters.

Reply:
- Does the user of NCurses have to pass an instance of Point when he call `add_char` | `print` ?